test/service/overseas.entities.service.spec.ts

```diff
@@ -63,3 +69,33 @@ describe('Overseas Entity Service test suite', () => {
   });
 
 });
+
+describe('Update Overseas Entity Service test suite', () => {
```

Comment:
Aren't we using double quotes for these?

Reply:
Better use template literals here.

Reply:
done